container-storage-setup.sh

```diff
@@ -0,0 +1,1286 @@
+#!/bin/bash
```

Comment:
Why this is showing as new file. It should have shown as simple rename. Like libcss.sh.